README.md

```diff
@@ -3,8 +3,12 @@
 This repository contains the capstone project for a pod of interns (step1-2020)
 in Google's Student Training in Engineering program.
 
+## Set Up
+
 This project requires JDK 11.
 
+This project also requires the an [API Key](https://developers.google.com/custom-search/v1/introduction) and three [Google Custom Search Engine IDs](https://cse.google.com/cse/all) added to the `SearchServlet.java` file per restricted site client: [StackOverflow](http://stackoverflow.com/), [W3Schools](https://www.w3schools.com/), [GeeksForGeeks](https://www.geeksforgeeks.org/).
```

Comment:
"requires an API key"

Reply:
Should it be "one CSE ID...per site client", not three? Might also want to clarify that the CSE needs to be set up to be restricted to the website.

Reply:
Stackoverflow link is not HTTPS